components/slices/TextSection.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+const TextSection = ({ slice }) => {
+  return (
+    <>
+      <h3>Text Section: </h3>
+      <p>{slice.primary.text[0].text}</p>
```

Comment:
Recommendation: 
- For text, use the prismic client rich text renderer from the start (though I see you are doing this already elsewhere). I recommend making your own RichText component that wraps the Prismic one to abstract away implementation details. 
- Keep the `slice.primary.text...` stuff out of the presentational component. You can format the props to be more friendly in the `getStaticProps` function. This just further abstracts away Prismic and future-proofs your site. You could define some more friendly props for TextSection and have SliceZone pass those in.